src/Network/Response.php

```diff
@@ -56,7 +56,7 @@ class Response
         303 => 'See Other',
         304 => 'Not Modified',
         305 => 'Use Proxy',
-        306 => 'Switch Proxy',
+        306 => '(Unused)',
```

Comment:
I can remove it. However I can't see the full picture if that's a good thing to do but rather keep it as it is. It is often not omitted but simply set to 'Unused', sometimes even the old message is there and its not clear that it should not be used.

It doesn't - however - exist on https://httpstatuses.com.
If you agree (just thumb up) I can remove 306.